Code/Framework/AzToolsFramework/Tests/PythonLoaderTests.cpp

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/Component/Entity.h>
+#include <AzCore/RTTI/BehaviorContext.h>
+#include <AzCore/Serialization/SerializeContext.h>
+#include <AzCore/std/containers/vector.h>
+#include <AzCore/std/sort.h>
+#include <AzTest/AzTest.h>
+#include <AzTest/Utils.h>
+#include <AzToolsFramework/Application/ToolsApplication.h>
+#include <AzToolsFramework/UnitTest/AzToolsFrameworkTestHelpers.h>
+#include <AzToolsFramework/Entity/EditorEntityContextComponent.h>
+#include <AzCore/UnitTest/TestTypes.h>
+#include <AzFramework/IO/LocalFileIO.h>
+#include <AzToolsFramework/API/PythonLoader.h>
+
+namespace UnitTest
+{
+    class AzToolsFrameworkPythonLoaderFixture
+        : public ToolsApplicationFixture<>
```

Comment:
The` ToolsApplicaitonFixture<>` is no longer needed and I believe a regular `LeakDetectionFixture` can be used, since there is no dependency on the FileIO system anymore.
```suggestion
    class AzToolsFrameworkPythonLoaderFixture
        : public LeakDetectionFixture
```

Reply:
There still is, i made a comment earlier about using FileIO's FindFiles vs. SystemFile's FileFiles. I wanted to use FileIO's version because it does a recursive search using FineFiles, but if I use SystemFile's, i'd have to manually implement the recursive search.

Reply:
Alternatively, I add a local `LocalFileIO` instance and use that directly